### DISCUSSION TITLE
update

### DISCUSSION THREAD

**Discussion by PaulRohlik** [frangoteam/FUXA#929] [Q&A]
Hello,

Please help. The update via the link is still version 1.1.14
Installation by other means will remove chart history.
What is the correct link please?

( npm install -g --unsafe-perm @frangoteam/fuxa )

**Comment by unocelli** [comment]
Hi, if you install from npm, from where you run fuxa 3 data folders are created : _appdata, _db, _logs, those folders are not deleted when you do the update. _db contains the daq (chart) history

**Reply by PaulRohlik** [reply]
Thanks :)
It works great. Great job
